src/cms/models/pages/page_translation.py

```diff
@@ -119,6 +119,20 @@ def combined_text(self):
             return attached_text + self.text
         return self.text + attached_text
 
+    @property
+    def combined_last_updated(self):
+        """
+        This function combines the last_updated date from this PageTranslation and from a mirrored page.
+        If this translation has no content, then the date from the mirrored translation will be used. In
+        other cases, the date from this translation will be used.
+
+        :return: The last_upated date of this or the mirrored page translation
+        :rtype: ~django.db.models.DateTimeField
+        """
+        if self.page.get_mirrored_page is not None and not self.text:
```

Comment:
```suggestion
        if self.page.get_mirrored_page and not self.text:
```
There is no way a foreign key contains something which evaluates to `False` but is not `None`, so we don't need to be that explicit here.